### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by Saladoc**
Some minor fixes I noticed while working on other stuff but found out of scope for the respective PRs

### CODE REVIEW

**Comment by Bastian** [comment]
Unused imports? O.o
Should our checkstyle checks detect  those?

**Comment by Saladoc** [comment]
It probably should, I've also added the check.

File: javacord-core/src/main/java/org/javacord/core/util/logging/ExceptionLoggerDelegateImpl.java [hunk 1]
```diff
@@ -41,9 +41,9 @@
     @Override
     public UncaughtExceptionHandler getUncaughtExceptionHandler() {
         return (thread, throwable) -> logger.error(
-                "Caught unhandled exception on thread '{}'!",
-                thread::getName,
-                () -> ExceptionLoggerDelegate.unwrapThrowable(throwable));
+                "Caught unhandled exception on thread '{}}'!",
```
Comment (Bastian) [inline javacord-core/src/main/java/org/javacord/core/util/logging/ExceptionLoggerDelegateImpl.java:44]: Was the second `}` intended?

Comment (Saladoc) [reply javacord-core/src/main/java/org/javacord/core/util/logging/ExceptionLoggerDelegateImpl.java:44]: It was not. IntelliJ autocompleted one while I typed the other.

**Comment by Bastian** [comment]
Thank you!

Fyi: The common convention for commit subjects is to not have a period at the end. See [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/). I don't enforce strict commit message rules for Javacord, but I thought you might be interested in it 🙂

**Pull request merged**